sdks/python/apache_beam/io/filebasedsource.py

```diff
@@ -235,11 +241,11 @@ class _SingleFileSource(iobase.BoundedSource):
 
   def __init__(self, file_based_source, file_name, start_offset, stop_offset,
                min_bundle_size=0, splittable=True):
-    if not isinstance(start_offset, integer_types):
+    if not isinstance(start_offset, (int, long)):
```

Comment:
If you import from `past` you can replace this with just `int`.

See http://python-future.org/compatible_idioms.html#long-integers